api/accounts/add.js

```diff
@@ -0,0 +1,20 @@
+module.exports = addSession
+
+var createAccount = require('../../utils/account/create')
+
+function addSession (state, options) {
+  return new Promise(function (resolve, reject) {
+    createAccount({
+      couchUrl: state.url,
+      username: options.username,
+      password: options.password,
+      includeProfile: options.include === 'account.profile'
```

Comment:
do you need to mention in [here](http://docs.accountrestapi.apiary.io/#reference/user/session/sign-out) that `?include=account.profile` is supported?